contrib/buildrefactor/src/python/pants/contrib/buildrefactor/buildozer.py

```diff
@@ -72,17 +72,31 @@ def _execute_buildozer_script(self, command):
       Buildozer.execute_binary(command, address.spec, binary=self._executable)
 
   @classmethod
-  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c'):
+  def execute_binary(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c', suppress_warnings=False):
     binary = binary if binary else BinaryUtil.Factory.create().select_binary('scripts/buildozer', version, 'buildozer')
+    Buildozer._execute_buildozer_command([binary, command, spec], suppress_warnings, output=False)
 
-    Buildozer._execute_buildozer_command([binary, command, spec])
+  @classmethod
+  def return_buildozer_output(cls, command, spec, binary=None, version='0.6.0.dce8b3c287652cbcaf43c8dd076b3f48c92ab44c', suppress_warnings=False):
+    binary = binary if binary else BinaryUtil.Factory.create().select_binary('scripts/buildozer', version, 'buildozer')
+    return Buildozer._execute_buildozer_command([binary, command, spec], suppress_warnings, output=True)
 
   @classmethod
-  def _execute_buildozer_command(cls, buildozer_command):
-    try:
-      subprocess.check_call(buildozer_command, cwd=get_buildroot())
-    except subprocess.CalledProcessError as err:
-      if err.returncode == 3:
-        logger.warn('{} ... no changes were made'.format(buildozer_command))
-      else:
-        raise TaskError('{} ... exited non-zero ({}).'.format(buildozer_command, err.returncode))
+  def _execute_buildozer_command(cls, buildozer_command, suppress_warnings, output):
+    if not output:
+      try:
+        subprocess.check_call(buildozer_command, cwd=get_buildroot())
+      except subprocess.CalledProcessError as err:
+        if err.returncode == 3:
+          if not suppress_warnings:
+            logger.warn('{} ... no changes were made'.format(buildozer_command))
+        else:
+          raise TaskError('{} ... exited non-zero ({}).'.format(buildozer_command, err.returncode))
+    else:
+      try:
+        subprocess.check_output(buildozer_command, cwd=get_buildroot(), stderr=subprocess.STDOUT)
```

Comment:
I did not get the chance to run the code step by step yet, so cannot tell the reason for `stderr=subprocess.STDOUT`, but I am guessing this method can be further simplified.

Reply:
@itsedvard lets coordinate on this as both the `print-target` and `meta-relocate` tasks are dependent on this refactoring.